.github/CHANGELOG.md

```diff
@@ -510,6 +512,16 @@
 
 <h3>Breaking changes</h3>
 
+* The `qml` module is no longer automatically imported with PennyLane and should instead be imported
+  separately using `import pennylane.qnn`. For example, `KerasLayer` can be accessed using:
+  
+  ```python
+  import pennylane as qml
+  import pennylane.qnn
+  
+  qlayer = qml.qnn.KerasLayer(...)
+  ```
+
```

Comment:
To confirm - we don't want to update mentions of `qml.qnn` in the changelog before this release?

Reply:
nope, changelog versions prior to v0.15-dev are immutable